tests/test_cudf.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+
+import cudf as gd
+import pandas as pd
+
+# For full unit tests of rapids cudf, please run the following commands after building the docker.
+# docker run --runtime nvidia --rm -it kaggle/python-gpu-build /bin/bash
+# export LD_LIBRARY_PATH=/usr/local/cuda/lib64
+# python -c "import cudf as gd"
+# conda uninstall --yes pytest && pip install -U pytest
+# cd /opt/conda/lib/python3.6/site-packages/cudf/tests
+# pytest -v --ignore=test_sparse_df.py --ignore=test_orc.py
+
+class TestCudf(unittest.TestCase):    
+    def test_read_csv(self):
```

Comment:
You need to add [`@gpu_test`](https://github.com/Kaggle/docker-python/blob/master/tests/test_nvidia.py#L12) to all these 3 tests, otherwise they are run against the CPU images which doesn't have these new packages.

Reply:
sure. Thank you for letting me know. Sorry for missing that decorator.

Reply:
I have made the changes and verified the tests in docker images. Thank you very much.